templates/commands/render_flags.go

```diff
@@ -122,6 +126,13 @@ func (r *RenderFlags) Register(set *cli.FlagSet) {
 		Usage: "Prompt the user for template inputs that weren't provided as flags.",
 	})
 
+	f.BoolVar(&cli.BoolVar{
```

Comment:
I wonder:

- Should this be under a new "TEMPLATE AUTHORS" or "DEVELOPMENT" flag header?
- Should this be hidden?

Reply:
Added a new section TEMPLATE AUTHORS as suggested.

I don't think we want it to be hidden; I think we want it to be reasonably discoverable by template authors.